checks/security_policy.go

```diff
@@ -44,9 +45,16 @@ func SecurityPolicy(c *checker.CheckRequest) checker.CheckResult {
 	}
 
 	// Set the raw results.
-	if c.RawResults != nil {
-		c.RawResults.SecurityPolicyResults = rawData
+	pRawResults := rawResults(c)
+	pRawResults.SecurityPolicyResults = rawData
```

Comment:
This is a race condition, see above

Reply:
Each check writes to a different part of the structure, like it was in the code before. No?